### DISCUSSION TITLE
Measurement Tool that uses travel multiplier

### DISCUSSION THREAD

**Discussion by MeusRex** [MeusRex/pf2e-chex#15] [Ideas]
- [ ] check if existing foundry toolscan be co-opted for this
- [ ] implement

**Comment by federicoacker** [comment]
There's this module which works with difficult terrain and makes calculations automatically that might give some inspiration
https://foundryvtt.com/packages/difficultterrain

**Reply by MeusRex** [reply]
Thank you, looks interesting and the code uses modules, multiple files, is clear and concise and it even has comments. Should be quite helpful, even if it is like 4 year old code.
